services/provider/proto/provider.proto

```diff
@@ -169,6 +169,7 @@ message ReportStatusRequest{
 }
 
 message ReportStatusResponse{
-    // Contains subscription channel of provider operator for client operator to match
-    string desiredClientOperatorChannel = 1;
+    // Contains hash of desired state of client
+    string desiredClientConfigHash = 1;
```

Comment:
1. Client config is a broad term that does not mean anything specifically and as such can create confusion. Add to that the fact that it does not contain a hash of the entire client configuration which means the name is also misleading.

2. There is no need to surface the fact that this is calculated as a hash as part of the API message definition. From the API perspective, it is an opaque value.

Reply:
what I can think of right now is `desiredClientState` wdyt?